Beepsky will now save the station by stuncuffing ops v3 FIX N' REBALANCE EDITION

Tacolizard opened the pull request:
:cl: Beepsky VS Ops
add: syndicate hardsuits and masks now affect securitron threat level
/:cl:

EVERYONE CALM DOWN OPS SUCKS ANYWAY AND WIZARD CLOTHES ALREADY AFFECT THREAT LEVEL.

ALSO ANY DECENT OP CAN SHOOT BEEPSKY LIKE ONCE AND KILL HIM.

Ok fine the agent card part was removed. now the **syndi gear just affects threat level the same as wizard gear. It's really not that weird.**

**The op must be wearing a syndie hardsuit, mask and have a weapon in-hand for beepsky to arrest them.**

Tacolizard committed:
Beepsky will now save the station by stuncuffing ops v2 DO-OVER EDITION

LanCartwright commented:
# No.

Tacolizard commented:
COBBY STOP USING EMOJIS TO ERASE MY SUPPORT

kevinz000 commented:
:-1:

Tacolizard commented:
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE


PLEASE MERGE

kevinz000 commented:
How about this. You play as an operative with 40TC, and I build 3 beepskies, and sic them on you. All the while 3 crewmembers are rushing you. See what happens.

kevinz000 commented:
Also, unnecessary Agent ID nerf, and mindshield implant buff. You're making anyone loyalty implanted immune to arrest status alone. Also, what if a friendly is wearing mask/hardsuit? Stop screwing with things.

kevinz000 commented:
Also >cheeremoting your own PR

Tacolizard committed:
Threat levels are now balanced so beepsky will only arrest ops with hardsuits+mask+weapon out or on belt.

Tacolizard commented:
@kevinz000 I actually forgot that beepsky only arrests threatlevels >=4 when i tweaked the mindshield part. i just updated it so that mindshields subtract 1 again. Also the reason why i removed agent cards from threatlevels was because beepsky wouldn't arrest an op if they had agent cards. Also it's been rebalanced so that syndie hardsuits and masks both increase threatlevel by 2, so a friendly with one of them won't be arrested.

PKPenguin321 commented:
:-1:

Shadowlight213 commented:
:-1: this nerfs regular traitors who use the agent card to escape THE LAW.

Tacolizard committed:
Ok have your agent cards back

Tacolizard commented:
@Shadowlight213 fine i readded the agent cards. Now ops must be wearing a mask+suit+have a weapon out or on belt.

TechnoAlchemisto commented:
I don't think you understand how much of a hassle it is to shoot beepsky

kevinz000 commented:
Ha lol it doesn't even compile we've got a @ma44 over here!

coiax commented:
I'd just like to mention that Beepsky threat levels are kinda obscure anyway, I always thought they were weird when looking at them in code.

Tacolizard committed:
should fix this so it compiles. it was a spacing problem

Tacolizard committed:
Should be the final rebalance of the PR.

Shadowlight213 commented:
It's not the same as wizard gear. The suit increases threat by 3 while the wizard hat increases by 2~~

Tacolizard commented:
@Shadowlight213 I realized i had to increase that because arrests only happen >=4, agent cards subtract 5, and having a suit and mask and weapon out would only have a net threat level of 3. I had to either nerf the agent cards or increase suit threat. I chose the latter because people seem to be very protective of Agent IDs

kevinz000 commented:
No

Shadowlight213 commented:
I was joking about the levels vs the wizard thing. It's fine.

Tacolizard commented:
well now i have a ton of downvotes from the original commit, so RIP me

optimumtact commented on code/modules/mob/living/carbon/human/human.dm:
```diff
@@ -583,14 +583,22 @@
 	if(isloyal(src))
 		threatcount -= 1
-
+		
```
can you delete this stray spacing please

Cheridan commented:
I'm sorry but it doesn't seem like this is a desired change at this time
